.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing to v8n
+
+- [Development Setup](#development-setup)
+- [Project Structure](#project-structure)
+
+## Development Setup
+
+You will need [Node.js](http://nodejs.org) **version 6+**.
+
+After cloning the repo, run:
+
+``` bash
+$ npm install # or yarn
+```
+
+If you use `npm` please do not commit the npm-lockfile. Keeping a lockfile for `yarn` and `npm` is not great to manage as versions may differ. ``package-lock.json`` is not specifically ignored, so please just do not add it to your commits.
```

Comment:
Better would be to just ignore the `package-lock.json`, since you are not depending on it anyway. Do make users think and mess with things for no reason.

Reply:
I agree, not my call on that. Just tried to include the current state here.